packages/ui-kit-components/src/components/Graph/graph.tsx

```diff
@@ -0,0 +1,238 @@
+import { Palette, useTheme } from '@mui/material';
```

Comment:
The color for the y-axis lines stays consistent across the alternating background colors in the designs.  But here, in both light and dark mode, that doesn't appear to be the case.

Reply:
It's still doing it.. The background is on top of the lines. I've put the lines in red & thicker for visibility

![image](https://user-images.githubusercontent.com/30781074/201977780-017e71b1-bb78-459b-9d1e-8582afbde10a.png)


Background at opacity 1

![image](https://user-images.githubusercontent.com/30781074/201977941-f2425ec5-18aa-4eff-9c23-759bd015ba8a.png)

Reply:
![Screen Shot 2022-11-15 at 12 10 46 PM](https://user-images.githubusercontent.com/106985561/201982891-d14044eb-dd02-4d26-bc72-7da0d82bb2b4.png)
![Screen Shot 2022-11-15 at 12 10 55 PM](https://user-images.githubusercontent.com/106985561/201982912-b3c588d3-b697-48c7-beef-143b74df690e.png)

Reply:
I've changed the background columns drawn before the axis.